### PR TITLE
Graph - add counters for removed labels and properties

### PR DESCRIPTION
**Pull request by DvirDukhan**
### Pull Request check-list

_Please make sure to review and check all of these items:_

- [ ] Does `$ tox` pass with this change (including linting)?
- [ ] Do the CI tests pass with this change (enable it first in your forked repo and wait for the github action build to finish)?
- [ ] Is the new or changed code fully tested?
- [ ] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?
- [ ] Is there an example added to the examples folder (if applicable)?
- [ ] Was the change added to CHANGES file?

_NOTE: these things are not required to open a PR and can be done
afterwards / while the PR is open._

### Description of change

This PR adds counters of removed labels and properties in graph result set statistics.
For those code changes to be applicable https://github.com/RedisGraph/RedisGraph/pull/2481 and https://github.com/RedisGraph/RedisGraph/pull/2379 should be merged into RedisGraph `master` branch

### CODE REVIEW

File: redis/commands/graph/query_result.py [hunk 1]
```diff
@@ -325,6 +329,10 @@ def _get_stat(self, stat):
     def labels_added(self):
         return self._get_stat(LABELS_ADDED)
 
+    @property
+    def labels_removed(self):
```
Comment (chayim) [inline redis/commands/graph/query_result.py:333]: Can we add docstrings to each of these

File: redis/commands/graph/query_result.py [hunk 2]
```diff
@@ -325,6 +329,10 @@ def _get_stat(self, stat):
...
+        return self._get_stat(LABELS_REMOVED)
```
Comment (chayim) [inline redis/commands/graph/query_result.py:334]: Even though they're "gimmes" we should have unit tests, covering each property - this way we don't break them in the future.

**Comment by DvirDukhan** [comment]
@dvora-h, there is a flakey test here in parts of the code I didn't touch

**Comment by dvora-h** [comment]
Yes, I know this flaky test... I re-run the tests

**Comment by DvirDukhan** [comment]
@dvora-h ready

**Comment by dvora-h** [comment]
@DvirDukhan Can you move the docstrings into the functions?

**Comment by DvirDukhan** [comment]
@dvora-h done

**Review by dvora-h — APPROVED** [review]

**Pull request merged**